src/bp-templates/bp-nouveau/buddypress/common/js-templates/activity/form.php

```diff
@@ -19,16 +19,21 @@
 		'parts/bp-activity-media',
 		'parts/bp-activity-document',
 		'parts/bp-activity-video',
-		'parts/bp-activity-post-form-avatar',
```

Comment:
@annrra What happens if someone overrides the templates in the child theme?

Reply:
@chetansatasiya templates can be overridden in child theme - there should be no issue with that

Reply:
@annrra I am saying here that if someone already overrided `parts/bp-activity-post-form-avatar` templates then what will happen? Can you test in theme and check does it working or breaking anything?

Reply:
@chetansatasiya 
Tested if parts/bp-activity-post-form-avatar is already overrided in child theme.
No errors or anything broken

Reply:
AWESOME